lms/static/sass/base/_animations.scss

```diff
@@ -258,6 +258,18 @@
   }
 }
 
+@mixin fade-transition {
+  -webkit-transition: all 0.2s ease-out;
+    -moz-transition: all 0.2s ease-out;
```

Comment:
Please using the predefined timing variables (https://github.com/edx/edx-platform/blob/mjevtic/SOL-531/lms%2Fstatic%2Fsass%2Fbase%2F_variables.scss#L46) when setting a transition time.